.github/template_specific/README_template.md

```diff
@@ -11,35 +11,56 @@ Short description of this module.
 ## Content
 1. [Compatibility](#compatibility)
 2. [Requirements](#requirements)
-    1. [Required software](#required-software)
-    2. [Other](#other)
-2. [Usage](#usage)
-    1. [Providers](#providers)
-3. [Inputs](#inputs)
-4. [Outputs](#outputs)
-5. [Example](#example)
+    1. [Required modules](#required-modules)
+    2. [Required software](#required-software)
+    3. [Other](#other)
+3. [Usage](#usage)
     1. [Verifying setup](#verifying-setup)
-6. [Contributors](#contributors)
-7. [License](#license)
+    2. [Providers](#providers)
+4. [Example usage](#example-usage)
+5. [Inputs](#inputs)
+6. [Outputs](#outputs)
+7. [Contributors](#contributors)
+8. [License](#license)
 
 ## Compatibility
-List of compatible versions. E.g.
-1. Terraform 0.12.2 or newer
-2. Nomad 0.12 or newer
+| Software | OSS Version | Enterprise Version |
+|:---|:---|:---|
+| Terraform | 0.13.1 or newer ||
+| Nomad | 0.12.3 or newer ||
+
+Any other?
```

Comment:
Should remove the questions before merging?

Reply:
These are questions for the module-creator to fill out. I think they are a good reminder.

Reply:
Oh, then I agree it should be there 👍